### ISSUE TITLE
CORS issue when referencing visio-web-embedded.js

### ISSUE THREAD

**Issue by patrikhellgren** [OfficeDev/office-js#1378]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

When referencing visio-web-embedded.js as an external library in a SPFx solution we get a CORS error "No 'Access-Control-Allow-Origin' header is present on the requested resource.". This was working up until about a week ago and no code changes has been done on our side.

## Expected Behavior
We should be able to use visio-web-embedded.js without CORS issues.

## Current Behavior
When referencing visio-web-embedded.js in the same way as in [https://github.com/pnp/sp-dev-fx-webparts/blob/master/samples/react-visio/config/config.json](https://github.com/pnp/sp-dev-fx-webparts/blob/master/samples/react-visio/config/config.json) we get the CORS error mentioned above.
This is also the documented way of loading an external library from a CDN: [https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/basics/add-an-external-library#load-a-script-from-a-cdn](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/basics/add-an-external-library#load-a-script-from-a-cdn).
We can also reproduce this by using [https://www.test-cors.org](https://www.test-cors.org/#?client_method=GET&client_credentials=false&server_url=https%3A%2F%2Fappsforoffice.microsoft.com%2Fembedded%2F1.0%2Fvisio-web-embedded.js&server_enable=true&server_status=200&server_credentials=false&server_tabs=remote).
By using PowerShell and running the following we can see that the host doesn't return any Access-Control-Allow-Origin header like it should:
`(Invoke-WebRequest https://appsforoffice.microsoft.com/embedded/1.0/visio-web-embedded.js -Method Options).Headers`

## Context
We have a webpart using visio-web-embedded.js live in production at multiple customers and it suddenly stopped working about a week ago. We were forced to build a new version with visio-web-embedded.js bundled in the package as a workaround which really not is an acceptable solution since the recommended way is to reference it from the URL mentioned in [https://docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/visio-javascript-reference-overview](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/visio-javascript-reference-overview).

## Your Environment
* Platform [PC desktop, Mac, iOS, Office on the web]: PC
* Operating System: Windows 10
* Browser (if using Office on the web): Tested in Edge, Chrome and IE 11 

## Useful logs
Access to XMLHttpRequest at 'https://appsforoffice.microsoft.com/embedded/1.0/visio-web-embedded.js' from origin 'https://www.test-cors.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by AlexJerabek** [comment]
Thanks for reporting this @patrikhellgren. 

@JuaneloJuanelo, could you please help answer this Visio question, or help find the right owner?

**Comment by ElizabethSamuel-MSFT** [comment]
@prateek9us Can you help with this issue? Thx.

**Comment by patrikhellgren** [comment]
Any news on this issue, it´s been almost two months?

**Comment by nbelyh** [comment]
@patrikhellgren There is probably nothing wrong with referencing Visio API using the standard office-js package. The point is when creating the sample in 2018, the office-js package was non-existing (for Visio). Now you can simply install the package, Visio API is included. Check out here the comment from the author of the sample you are using as a reference: https://github.com/pnp/sp-dev-fx-webparts/issues/1629#issuecomment-739061475

**Comment by patrikhellgren** [comment]
@nbelyh Thanks for the heads up. I will give that a try. It's just unfortunate that this is so undocumented and that the existing documentation doesn't mention any of this and instead mentions the reference way as the way to do it.

**Comment by nbelyh** [comment]
@patrikhellgren  Looks like visio online api is not the piority number one for now 😅 .unfortunately I also faced a number of issues, this one for  example can become blocker, if you want to use this api actively  https://github.com/OfficeDev/office-js/issues/1539 😞
